site/data/data.json

```diff
@@ -19664,6 +19964,21 @@
       }
     ]
   },
+  {
+    "name": "Spotted Owl And L'Abatros",
```

Comment:
I hid this entry, added Kindra to the Spotted Owl entry

Reply:
Perfect. You can actually delete it all together, the script won't re-add the same person regardless of what their place name is.